README.md

```diff
@@ -29,6 +29,12 @@ cargo install cargo-llvm-cov
 cargo llvm-cov
 ```
 
+## Coverage Report
+
+```bash
+cargo llvm-cov --html or cargo llvm-cov --open
+```
+
```

Comment:
```suggestion
The finished report can be found at `target/llvm-cov/html/`
```

the report tells you where you can find it when it's done, but I think it'd also be nice to have it here

Reply:
Since the `cargo llvm-cov --html` command will display `Finished report saved to /Users/user_path/axelar-cgp-soroban/target/llvm-cov/html` after it's completion, I think it's not necessary